homeassistant/components/airtouch4/climate.py

```diff
@@ -98,28 +98,20 @@ def __init__(self, coordinator, ac_number, info):
         self._ac_number = ac_number
         self._airtouch = coordinator.airtouch
         self._info = info
-        self._unit = self._airtouch.GetAcs()[self._ac_number]
+        self._unit = self._airtouch.GetAcs()[ac_number]
+        self._attr_unique_id = f"ac_{ac_number}"
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, f"ac_{ac_number}")},
```

Comment:
Is this identifier unique enough to not conflict if there are multiple config entries?

Reply:
This is an existing issue though

Reply:
Oh apparently the GitHub account of the code owner is gone as well. Guess we will never know